### PR TITLE
polish docs

### PR DESCRIPTION
**Pull request by zoepiran**

### CODE REVIEW

**Review by MUCDK — APPROVED** [review]

**Review by MUCDK — CHANGES_REQUESTED** [review]
see comments

**Comment by MUCDK** [comment]
@giovp can you check regarding "best" use of squidpy please?

File: tutorials/500_spatiotemporal.ipynb [hunk 1]
```diff
@@ -7,23 +7,43 @@
    "source": [
```
Comment (MUCDK) [inline tutorials/500_spatiotemporal.ipynb:7]: <ul><li>typo <code>knowledge</code>,would prefer another term though, e.g. <code>...across time incorporating spatial information</code>.</li><li>only introduce symbols/parameters which you need later explicitly (e.g. in a formula), if you don't use the C_x, C_y, C, I think it's not necessary to have them here, whereas it's fair to keep alpha and epsilon, as these are parameters which need to be explained, as you use them in the <code>solve</code> method </li><li><br></li></ul><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (MUCDK) [inline tutorials/500_spatiotemporal.ipynb:7]: <p>twice <code>spatiotemporal transcriptomics atlas</code> , rephrahse/remove </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (MUCDK) [inline tutorials/500_spatiotemporal.ipynb:7]: <p>Do we want to keep all these variables, e.g. why do we have obs columns of the form <code>{celltype}_annotation</code>,also not sure we need all of the <code>_colors</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (MUCDK) [inline tutorials/500_spatiotemporal.ipynb:7]: <p>Don't we want to use <code>score_genes_for_marginals</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (MUCDK) [inline tutorials/500_spatiotemporal.ipynb:7]: <p>type in <code>epsilon=0</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (zoepiran) [reply tutorials/500_spatiotemporal.ipynb:7]: <p>okay. other tutorials shall be modified accordingly as well.</p>

Comment (zoepiran) [reply tutorials/500_spatiotemporal.ipynb:7]: <p>its not necessary to save all - it was for the per-organ mapping. guess the object can be modified.</p>

Comment (zoepiran) [reply tutorials/500_spatiotemporal.ipynb:7]: <p>i think we can avoid it here and keep the results as is. We have other examples using it.</p>

Comment (zoepiran) [reply tutorials/500_spatiotemporal.ipynb:7]: <p>and the reproducibility repo to recover results</p>

Comment (MUCDK) [reply tutorials/500_spatiotemporal.ipynb:7]: Which notebook are you talking about?

Comment (MUCDK) [reply tutorials/500_spatiotemporal.ipynb:7]: The results we show in the paper make use of it. So I would also add it here tbh.

Comment (MUCDK) [reply tutorials/500_spatiotemporal.ipynb:7]: I would remove the unnecessary data from the anndata object.

Comment (zoepiran) [reply tutorials/500_spatiotemporal.ipynb]: `100_lineage.ipynb`
and
`200_temporal_problem.ipynb`

Comment (zoepiran) [reply tutorials/500_spatiotemporal.ipynb]: sharing the updated object for you to upload

**Comment by giovp** [comment]
I think it looks good,. In paper, we leverage the fact that the spatial coordinates are already basically arranged so that without library key the points are still arranged properly. i think it's fine either way!

**Pull request merged**